zerver/openapi/zulip.yaml

```diff
@@ -805,160 +817,7 @@ paths:
-          Whether to send breadcrumb message to the old thread to
-          notify users where the messages were moved to.
-
-          **Changes**: New in Zulip 3.0 (feature level 9).
-        schema:
-          type: boolean
-          default: true
-        example: true
-      - name: send_notification_to_new_thread
-        in: query
-        description: |
-          Whether to send a notification message to the new thread to
-          notify users where the messages came from.
-
-          **Changes**: New in Zulip 3.0 (feature level 9).
-        schema:
-          type: boolean
-          default: true
-        example: true
-      - $ref: '#/components/parameters/Content'
-      - $ref: '#/components/parameters/StreamIdInQuery'
-      responses:
-        '200':
-          description: Success.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/JsonSuccess'
-        '400':
-          description: Bad request.
-          content:
-            application/json:
-              schema:
-                allOf:
-                - $ref: '#/components/schemas/CodedError'
-                - properties:
-                    msg:
-                      enum:
-                      - Your organization has turned off message editing
-                      - You don't have permission to edit this message
-                      - The time limit for editing this message has past
-                      - Nothing to change
-                      - Topic can't be empty
-                - example:
-                    {
-                        "code": "BAD_REQUEST",
-                        "msg": "You don't have permission to edit this message",
-                        "result": "error"
-                    }
-    delete:
-      operationId: delete_message
-      tags: ["messages"]
-      description: |
-        Permanently delete a message.
 
-        `DELETE {{ api_url }}/v1/messages/{msg_id}`
-
-        This API corresponds to the
-        [delete a message completely][delete-completely] feature documented in
-        the Zulip Help Center.
-      parameters:
-      - $ref: '#/components/parameters/MessageId'
-      responses:
-        '200':
-          description: Success.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/JsonSuccess'
-        '400':
-          description: Bad request.
-          content:
-            application/json:
-              schema:
-                oneOf:
-                - $ref: '#/components/schemas/InvalidMessageError'
-                - allOf:
-                  - $ref: '#/components/schemas/CodedError'
-                  - example:
-                      {
-                          "code": "BAD_REQUEST",
-                          "msg": "You don't have permission to delete this message",
-                          "result": "error"
-                      }
```

Comment:
Can you split the `zulip.yaml` changes into several commits that each explain what they're doing?

There's a lot of moving content around here.

Reply:
Done